includes/rest.php

```diff
@@ -27,14 +31,34 @@ function rest_routes() {
 			'permission_callback' => '__return_true',
 		)
 	);
+
+	register_rest_route(
+		'icw/v1',
+		'/count-for-fix/',
+		array(
+			'methods'             => 'POST',
+			'callback'            => __NAMESPACE__ . '\\count_for_fix_endpoint',
+			'permission_callback' => '__return_true',
+		)
+	);
+
+	register_rest_route(
+		'icw/v1',
+		'/fix/',
+		array(
+			'methods'             => 'POST',
+			'callback'            => __NAMESPACE__ . '\\fix_endpoint',
+			'permission_callback' => '__return_true',
```

Comment:
```suggestion
			'permission_callback' => 'edit_posts',
```

This might end up needing to be a little more complex to allow for CPTs registered with custom capabilities.

As it's editing the posts, it will definitely need to include a capability check and should probably include a nonce check for users without an application password.

Reply:
Should I take the time to research this now?